lib/spack/spack/build_systems/autotools.py

```diff
@@ -288,5 +291,14 @@ def check(self):
             self._if_make_target_execute('test')
             self._if_make_target_execute('check')
 
+    run_after('install')(PackageBase._run_default_install_time_test_callbacks)
+
+    def installcheck(self):
+        """Searches the Makefile for an ``installcheck`` target
+        and runs it if found.
+        """
+        with working_dir(self.build_directory()):
```

Comment:
@adamjstewart I think it should read `self.build_directory` as it is a property now.